controllers/nginx/rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -365,6 +372,12 @@ http {
             access_log off;
             return 200;
         }
+
+        {{ if $cfg.UseUpstreamHealthChecks }}
+        location /upstream_status {
```

Comment:
Please add an example, as in this pr: https://github.com/kubernetes/ingress/pull/235, there's already an examples/health-checks directory, just create a gce/ and nginx/ under that and add it under nginx/. 

Please also mention in that example what this endpoint exposes.